<ui version="4.0" >
 <class>KWin::CubeEffectConfigForm</class>
 <widget class="QWidget" name="KWin::CubeEffectConfigForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>566</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Tab 1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5" >
       <item row="1" column="1" >
        <widget class="QGroupBox" name="groupBox_5" >
         <property name="title" >
          <string>Background</string>
         </property>
         <layout class="QGridLayout" name="gridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_6" >
            <property name="text" >
             <string>Background Color:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy" >
             <cstring>backgroundColorButton</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="KColorButton" name="backgroundColorButton" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Wallpaper:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy" >
             <cstring>wallpaperRequester</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="KUrlRequester" name="wallpaperRequester" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0" colspan="2" >
        <widget class="QGroupBox" name="groupBox_8" >
         <property name="title" >
          <string>Activation</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3" >
          <item row="0" column="0" colspan="2" >
           <widget class="KWin::GlobalShortcutsEditor" native="1" name="editor" >
            <property name="minimumSize" >
             <size>
              <width>0</width>
              <height>200</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item rowspan="2" row="0" column="0" >
        <widget class="QGroupBox" name="groupBox_7" >
         <property name="title" >
          <string>Appearance</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2" >
          <item row="0" column="0" colspan="2" >
           <widget class="QCheckBox" name="displayDesktopNameBox" >
            <property name="text" >
             <string>Display desktop name</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2" >
           <widget class="QCheckBox" name="reflectionBox" >
            <property name="text" >
             <string>Reflection</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Rotation duration:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy" >
             <cstring>rotationDurationSpin</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QSpinBox" name="rotationDurationSpin" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="specialValueText" >
             <string>Default</string>
            </property>
            <property name="suffix" >
             <string> msec</string>
            </property>
            <property name="maximum" >
             <number>5000</number>
            </property>
            <property name="singleStep" >
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2" >
           <spacer name="verticalSpacer_2" >
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QGroupBox" name="groupBox_4" >
         <property name="title" >
          <string>Opacity</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6" >
          <item row="1" column="0" colspan="2" >
           <widget class="QSlider" name="cubeOpacitySlider" >
            <property name="minimumSize" >
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximum" >
             <number>100</number>
            </property>
            <property name="singleStep" >
             <number>1</number>
            </property>
            <property name="value" >
             <number>100</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition" >
             <enum>QSlider::TicksBelow</enum>
            </property>
            <property name="tickInterval" >
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <widget class="QSpinBox" name="cubeOpacitySpin" >
            <property name="minimumSize" >
             <size>
              <width>75</width>
              <height>0</height>
             </size>
            </property>
            <property name="suffix" >
             <string> %</string>
            </property>
            <property name="maximum" >
             <number>100</number>
            </property>
            <property name="value" >
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Transparent</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QLabel" name="label_5" >
            <property name="text" >
             <string>Opaque</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0" colspan="2" >
        <spacer name="verticalSpacer_3" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Tab 2</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7" >
       <item row="0" column="0" >
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Caps</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4" >
          <item row="0" column="0" colspan="2" >
           <widget class="QCheckBox" name="cubeCapsBox" >
            <property name="text" >
             <string>Show caps</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="capColorLabel" >
            <property name="text" >
             <string>Cap Color:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy" >
             <cstring>capColorButton</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="KColorButton" name="capColorButton" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2" >
           <widget class="QCheckBox" name="capsImageBox" >
            <property name="text" >
             <string>Display image on caps</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2" >
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Zoom</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8" >
          <item row="1" column="0" >
           <widget class="QLabel" name="label_8" >
            <property name="text" >
             <string>Near</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLabel" name="label_9" >
            <property name="text" >
             <string>Far</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2" >
           <widget class="QSlider" name="zPositionSlider" >
            <property name="toolTip" >
             <string>Define how far away the object should appear</string>
            </property>
            <property name="maximum" >
             <number>3000</number>
            </property>
            <property name="singleStep" >
             <number>10</number>
            </property>
            <property name="pageStep" >
             <number>100</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition" >
             <enum>QSlider::TicksBelow</enum>
            </property>
            <property name="tickInterval" >
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0" colspan="2" >
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1" >
        <widget class="QGroupBox" name="groupBox_9" >
         <property name="title" >
          <string>Additional options</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout" >
          <item>
           <widget class="QCheckBox" name="animateDesktopChangeBox" >
            <property name="toolTip" >
             <string/>
            </property>
            <property name="text" >
             <string>Display the cube when switching desktops</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="bigCubeBox" >
            <property name="toolTip" >
             <string>If enabled span the cube across all screens on a multiscreen setup, otherwise
the cube will only be displayed on a single screen</string>
            </property>
            <property name="text" >
             <string>Span cube on all screens in multi screen setup</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="closeOnMouseReleaseBox" >
            <property name="toolTip" >
             <string>If enabled the effect will be deactivated after rotating the cube with the mouse,
otherwise it will remain active</string>
            </property>
            <property name="text" >
             <string>Close after mouse dragging</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="walkThroughDesktopBox" >
            <property name="text" >
             <string>Use this effect for walking through the desktops</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KWin::GlobalShortcutsEditor</class>
   <extends>QWidget</extends>
   <header location="global" >kwineffects.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>displayDesktopNameBox</tabstop>
  <tabstop>reflectionBox</tabstop>
  <tabstop>rotationDurationSpin</tabstop>
  <tabstop>cubeOpacitySlider</tabstop>
  <tabstop>cubeOpacitySpin</tabstop>
  <tabstop>backgroundColorButton</tabstop>
  <tabstop>wallpaperRequester</tabstop>
  <tabstop>cubeCapsBox</tabstop>
  <tabstop>capColorButton</tabstop>
  <tabstop>capsImageBox</tabstop>
  <tabstop>animateDesktopChangeBox</tabstop>
  <tabstop>bigCubeBox</tabstop>
  <tabstop>closeOnMouseReleaseBox</tabstop>
  <tabstop>walkThroughDesktopBox</tabstop>
  <tabstop>zPositionSlider</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cubeOpacitySpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>cubeOpacitySlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>386</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel" >
     <x>235</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cubeOpacitySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>cubeOpacitySpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>133</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel" >
     <x>386</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
